pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.coverage.paths]
+source = ["src", "*/site-packages"]
+
+[tool.coverage.run]
+branch = true
+relative_files = true
+
+[tool.coverage.report]
+show_missing = true
+exclude_lines = ["if __name__ == .__main__.:", "pragma: no-cover", "@abstract", "def __repr__"]
+
+[tool.black]
+line-length = 88
+target-version = ['py38']
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+line_length = 88
```

Comment:
@djhoese It's not clear to me why we need all these?
Can we keep the PR minimal to cover only the Cython case?

Reply:
@AndreMiras From fwhat I can tell it looks like @miurahr tried to move some/all of the configuration in setup.cfg while at the same time adding a `build-system` section so this github action could be built as a python package maybe? It also looks like they added default configurations for black and how coverage for this project itself is reported. None of this requires that black is run, but the coverage configuration might need to be up to you if you agree with the lines that are excluded.

If you'd like I can revert all of the pyproject.toml changes that aren't needed and see how things behave.

Reply:
Yes I saw some of the config were indeed just moved around, but other got added and I'm not too sure about it, like `exclude_lines` indeed.
Could you PR only your changes if that's enough to cover your case?
Maybe you can amend the commits to recycle this PR if you know how to.
We don't need the whole back and forth history, so it could look like:
```
git reset --hard upstream/develop
git cherry-pick f3f66e3
git commit --amend
git push -f origin feature-cython-coverage
```
Or simply close and open an new one.
I leave that to you